src/org/labkey/test/tests/component/GridPanelTest.java

```diff
@@ -1088,7 +1090,7 @@ public void testSearchAndFilter() throws IOException, CommandException
 
         // Again can use a query to get the expected number of rows from the search.
         filters = new ArrayList<>();
-        filters.add(new Filter(FILTER_STRING_COL, searchString, Filter.Operator.getOperator("CONTAINS")));
+        filters.add(new Filter(FILTER_STRING_COL.getFieldKey().toString(), searchString, Filter.Operator.getOperator("CONTAINS")));
```

Comment:
`FieldInfo.toString()` is equivalent to `FieldInfo.getFieldKey().toString()`
```suggestion
        filters.add(new Filter(FILTER_STRING_COL.toString(), searchString, Filter.Operator.getOperator("CONTAINS")));
```

Reply:
Done